variables.tf

```diff
@@ -938,6 +944,16 @@ variable "enable_gatekeeper" {
 }
 
 variable "gatekeeeper_version" {
```

Comment:
Please remove it. It will only destroy a null_resource. The installation would already have been done if enabled.